functionapp.tf

```
@@ -0,0 +1,101 @@
+
+resource "azurerm_app_service_plan" "asp" {
+  location            = var.location
+  name                = format("%s-%s-%s", var.product, var.env, "asp")
+  resource_group_name = azurerm_resource_group.rg.name
+  kind                = "FunctionApp"
+  reserved            = false
+  sku {
+    size = var.aspskusize
```

use _ in between words in terraform

```suggestion
    size = var.asp_sku_size
```